engine/runtime-compiler/src/main/scala/org/enso/compiler/data/BindingsMap.scala

```diff
@@ -54,18 +82,18 @@ case class BindingsMap(
   override def restoreFromSerialization(
     compiler: Compiler
   ): Option[BindingsMap] = {
-    val packageRepository = compiler.getPackageRepository
-    this.toConcrete(packageRepository.getModuleMap)
+    this.pendingRepository = compiler.getPackageRepository
+    Some(this)
```

Comment:
This is potentially dangerous change. Previously, when the `restoreFromSerialization` failed, it would return `None` and that would _trigger parsing_. Right now we optimistically claim everything is all right and then we may get failures later when someone calls for example `ResolvedModule.unsafeModule`.

It is probably OK, but we should get ready for such reports in the future and fix them when they appear.